static/app/views/dashboardsV2/manage/index.tsx

```diff
@@ -126,6 +137,17 @@ class ManageDashboards extends AsyncView<Props, State> {
             />
             <PageHeader>
               <PageHeading>{t('Manage Dashboards')}</PageHeading>
+              <Button
```

Comment:
I think when we decide to move the dashboard manager as the landing page, we would only do it for orgs that have 'dashbobard-edit' feature enabled and those with the basic dashboard would just see the default overview. So I don't need to put this button behind the 'dashbobard-edit' flag right?

Reply:
@dashed might know more?